File: Programetions/Segundo modulo/aula3/atividade3.py
# - Crie uma função soma que receba dois números como parâmetros e retorne a soma deles.

numero1 = int(input('Insira o primeiro numero: '))
numero2 = int(input('Insira o segundo numero: '))

def somar():
    soma = numero1 + numero2
    return soma

print(somar())

File: Programetions/Quarto modulo/aula5/resposta_condicionais/atividade3.js
const numero1 = 1
const numero2 = 2
const numero3 = 3

if ((numero1 > numero2) && (numero1 > numero3)) {
    console.log(`O número ${numero1} é o número maior`);
} else if ((numero2 > numero1) && (numero2 > numero3)) {
    console.log(`O número ${numero2} é o número maior`);
} else {
    console.log(`O número ${numero3} é o número maior`);
}